dev/dll/Microsoft.UI.Xaml.vcxproj.filters

```diff
@@ -294,6 +294,78 @@
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
+    <Page Include="@(PageRequiringCustomCompilation)" />
+    <Page Include="@(PageRequiringCustomCompilation)" />
```

Comment:
Arg this is happening again. Please revert.

Reply:
Hmm. thought we fixed this a while back. Lets follow up in a separate change.